sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/ChatMessage.java

```diff
@@ -4,7 +4,7 @@
 
 package com.azure.communication.chat.models;
 
-import com.azure.communication.common.CommunicationUser;
+import com.azure.communication.common.CommunicationUserIdentifier;
```

Comment:
This is generated code. Is this the result of swagger file change and regeneration or a manual change?

Reply:
@JianpingChen This is actually not generated code, we're just adding these new classes. All generated code is in the implementation folder.